### ISSUE TITLE
Migrate user stories to Github issues

### ISSUE THREAD

**Issue by oneillkza** [NCBI-Hackathons/Epigenomics_CWL#27]
Github issues provide a lot more flexibility and functionality for tracking user stories than simple markdown like we've been using. I'd like to migrate our existing user stories into issues (ie copy/paste and then delete the markdown), and use issues moving forward.

**Comment by neksa** [comment]
I totally agree

**Comment by bdecato** [comment]
Agreed. I should have some time tomorrow afternoon to migrate existing stories. I'll also double check that there is a story for each tool on the workflow diagram as it currently exists.
